waku/node/v2/waku_types.nim

```diff
@@ -61,12 +61,20 @@ type
   MessagePush* = object
     messages*: seq[WakuMessage]
 
+  FilterRPC* = object
```

Comment:
This is not in spec is it? If you want to change the spec I'd create a spec PR to begin with

Reply:
Considering things still don't work and this improves things and that spec is in raw I'll approve PR, but we should get into habit of having at least a spec issue before we make spec changes

Reply:
@oskarth yeah will get the spec done. My main priority was getting this merged so we can actually use filter protocol